lib/spack/spack/cmd/help.py

```diff
@@ -47,6 +47,7 @@
       @r{-variant} or @r{~variant}          disable <variant>
       @B{variant=value}                 set non-boolean <variant> to <value>
       @B{variant=value1,value2,value3}  set multi-value <variant> values
+      @B{variant:=value1,value2}        set <variant> to ONLY value1 and value2
```

Comment:
Maybe something like

```
      @B{variant=value1,value2,value3}  <variant> includes these values (at least)
      @B{variant:=value1,value2}        <variant> has exactly these values
```

I'm not sure if I would understand the difference without prior knowledge.